test/end-to-end/core.sh

```diff
@@ -42,8 +42,14 @@ function wait_for_app() {
   wait_for_url_timed "http://${FRONTEND_IP}:5432" "[INFO] Frontend says: " $((2*TIME_MIN))
 
   echo "[INFO] Testing app"
+  set -x
+  OLD=$OS_USE_STACK_TRACE
+  OS_USE_STACK_TRACE=""
   wait_for_command '[[ "$(curl -s -X POST http://${FRONTEND_IP}:5432/keys/foo -d value=1337)" = "Key created" ]]'
```

Comment:
In an orthogonal direction, @deads2k do you remember why you argued these should *not* be refactored into `os::cmd::try_until_text` ?